tests/DBALTestConnectionTrait.php

```diff
@@ -27,10 +34,21 @@ protected function initializeConnection(): void
             );
```

Comment:
On line 31 there is still a check for sqlite? I think this is no longer needed/should be changed to MySQL?